packages/guides-cli/src/DependencyInjection/ContainerFactory.php

```diff
@@ -54,7 +54,7 @@ public function loadExtensionConfig(string $extension, array $config): void
 
         $extensionAlias = $this->registeredExtensions[$extensionFqcn] ?? false;
         if (!$extensionAlias) {
-            $this->registerExtension(new $extensionFqcn());
+            $this->registerExtension(new $extensionFqcn(), $config);
```

Comment:
@wouterj I think you missed this when implementing the extension config? Otherwise only default extensions can be configured, not the ones configured in the guides.xml?

Reply:
You're right :+1: 

I think I originally intended this without the early return on the next line (so we "register" the extension without config, and then add extra config). But this works as well.